src/main/java/com/auth0/TransientCookieStore.java

```diff
@@ -0,0 +1,154 @@
+package com.auth0;
+
+import org.apache.commons.lang3.Validate;
+
+import javax.servlet.http.Cookie;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import java.util.Arrays;
+import java.util.List;
+
+/**
+ * Allows storage and retrieval/removal of cookies.
+ */
+class TransientCookieStore {
+
+    private static final int MAX_AGE_SECONDS = 600; // 10 minutes
+
+    // Prevent instantiation
+    private TransientCookieStore() {}
```

Comment:
maybe, use abstract

Reply:
I'd prefer to not use abstract classes unless it is being used for actual inheritance purposes. Since this class exports only static methods, the private constructor just ensures no instances can be created since it doesn't make sense.